docs/mine-hnt/validators/mainnet/deployment-guide.mdx

```diff
@@ -131,11 +131,11 @@ package by PaulVMo, is the least complex to deploy and has very high performance
 has the highest level of complexity to deploy, requires a higher level of Linux technical expertise,
 and also has very high performance. The docker version is less complex to deploy than Source, if you
 already have experience with Docker, but does not perform as well as the other two options. The
-debian package and docker options are also the simplest to deploy and maintain when running multiple
-validator instances on a single host.
+Debian package is also the simplest to deploy and maintain when running multiple validator instances on a single host.
```

Comment:
Is this true? I'd expect Docker to be the way to manage multiple on a single host.

Reply:
Yes, based on how Paul configured the deb packages, it automatically increments the port mappings for both 215x and 808x for each instance, which removes all the confusion around configuring port mappings for each docker instance, which often leads to support questions in discord for people who aren't experts in docker.